contracts/governance/extensions/GovernorTimelockControl.sol

```diff
@@ -39,27 +39,17 @@ abstract contract GovernorTimelockControl is IGovernorTimelock, Governor {
     }
 
     /**
-     * @dev See {IERC165-supportsInterface}.
+     * @dev Overridden version of the {Governor-state} function status check on the timelock if in ss`Queued`.
```

Comment:
Not sure what was the intention here but it needs attention.

Reply:
Fixed.